recipes/ogre/1.x/conanfile.py

```diff
@@ -0,0 +1,285 @@
+import os
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import conan.tools.files
+import textwrap, shutil
+
+class OgreCmakeConan(ConanFile):
+    name = "ogre"
+    license = "MIT"
+    homepage = "https://github.com/OGRECave/ogre"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "A scene-oriented, flexible 3D engine written in C++ "
+    topics = ("graphics", "rendering", "engine", "c++")
+
+    settings = "os", "compiler", "build_type", "arch"
+
+    generators = "cmake", "cmake_find_package"
+    exports_sources = "CMakeLists.txt", "patches/**"
+    #options copied from https://github.com/StatelessStudio/ogre-conan/blob/master/conanfile.py
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "cp_bin_dir": "ANY",
+        "cp_media_dir": "ANY",
+        "disable_plugins": [True, False],
+        "OGRE_ASSERT_MODE": "ANY",
```

Comment:
please, keep options in lower-case, it's better because you keep a pattern and they are case-sensitive.

Reply:
done